circle.yml

```diff
@@ -7,9 +7,6 @@ machine:
 
 ## Customize dependencies..
 dependencies:
```

Comment:
That's not _quite_ what's going on here.  I'm just stopping circle from doing that step, because we're now using tox to run tests on python 2 & python 3.

tox takes care of making a venv for python 2 & 3 separately, and then installing the deps into each.  You can see it in action here: https://circleci.com/gh/rolepoint/flump/117